scripts/bfasst/transform/xilinx_phys_netlist.py

```diff
@@ -0,0 +1,161 @@
+""" Creates a xilinx netlist that has only physical primitives"""
+
+
+import re
+import subprocess
+import jpype
+import jpype.imports
+
+from bfasst.config import VIVADO_BIN_PATH
+from bfasst.paths import THIRD_PARTY_PATH
+from bfasst.status import Status, TransformStatus
+from bfasst.transform.base import TransformTool
+
+jpype.startJVM(classpath=[str(THIRD_PARTY_PATH / "rapidwright-2022.2.1-standalone-lin64.jar")])
```

Comment:
```suggestion
jpype.startJVM(classpath=[str(THIRD_PARTY_PATH / "rapidwright/bin")])
```
This would allow you to use the class files directly, I think...

Reply:
I'll give it a try

Reply:
Just for reference, this comes from:
https://docs.oracle.com/javase/1.5.0/docs/tooldocs/findingclasses.html
> A class file has a subpath name that reflects the class's fully-qualified name. For example, if the class `com.mypackage.MyClass` is stored under `/myclasses`, then `/myclasses` must be in the user class path and the full path to the class file must be `/myclasses/com/mypackage/MyClass.class`. If the class is stored in an archive named `myclasses.jar`, then `myclasses.jar` must be in the user class path, and the class file must be stored in the archive as `com/mypackage/MyClass.class`.

Reply:
This worked once I also included both the compiled files (/bin) and the distributed precompiled binaries (/jars)